### PR TITLE
Adding ability to set apns-collapse-id to iOS notifications

### PR DESCRIPTION
**Pull request by Ariandr**
Hi @onmyway133 
I implemented the ability to set `apns-collapse-id` to the notification header. Manually tested, works okay.
Solves the issue https://github.com/onmyway133/PushNotifications/issues/16

Also, just bumped the version to 1.6.0 and updated the description of the app.

Also fixed https://github.com/onmyway133/PushNotifications/issues/4 (selecting last chosen auth method for iOS)

Also fixed https://github.com/onmyway133/PushNotifications/issues/15. You initially fixed it, but there was no `return` keyword in `catch` block, so the request was sent with wrong parameters and `Success` message was displayed. Now it just doesn't send the request at all and displays the error message. E.g., `Failed: SyntaxError: Unexpected end of JSON input`

Screen Shot 2019-08-28 at 19 41 44

### CODE REVIEW

File: components/InputComponent.js [hunk 1]
```diff
@@ -142,13 +142,19 @@ class InputComponent extends React.Component {
...
+    }
+
     try {
       notification.rawPayload = JSON.parse(input.message)
     } catch(e) {
       this.props.updateOutput({
         loading: false,
         text: 'Failed: ' + e
       })
+      return
```
Comment (onmyway133) [inline components/InputComponent.js]: 👍 nice catch

File: components/InputComponent.js [hunk 2]
```diff
@@ -142,13 +142,19 @@ class InputComponent extends React.Component {
     const notification = new APN.Notification()
     notification.expiry = Math.floor(Date.now() / 1000) + 3600
 
+    const collapseId = input.collapseId
+    if (collapseId && collapseId != '') {
```
Comment (onmyway133) [inline components/InputComponent.js]: Maybe it's safer to use `!==`?

**Comment by onmyway133** [comment]
@Ariandr Awesome 💪

File: components/InputComponent.js (hunk 2 above)
Comment (Ariandr) [reply components/InputComponent.js]: Sure, done!

**Comment by Ariandr** [comment]
@onmyway133 
If you think that everything is okay, you can merge the request and create a new release 1.6.0. I will attach build artifacts to it 🙂

**Comment by onmyway133** [comment]
@Ariandr ja, here it is https://github.com/onmyway133/PushNotifications/releases/tag/1.6.0

**Comment by Ariandr** [comment]
Hi @onmyway133 
There was a request to update the App Store version of the app to version 1.6.0. I had a conversation with a user here: https://github.com/onmyway133/PushNotifications/issues/23

**Pull request merged**